src/translations/pages/assistant.ts

```diff
@@ -70,6 +70,13 @@ const AssistantInternal = {
       },
     },
   },
+  ticketBooking: {
+    globalMessage: _(
+      'Reisen krever billett som må kjøpes fra et annet selskap enn AtB.',
+      'This journey requires a ticket that must be purchased from provider other than AtB.',
```

Comment:
Should there be an "a" before "provider"? I.e. 'This journey requires a ticket that must be purchased from a provider other than AtB.'

Reply:
Fixed some minor errors now. The text should be the same as in the app.